configuration/forms.md

```diff
@@ -529,6 +530,18 @@ _When `xml2sms` is not set or falsy, ODK compact record representation will be u
 
 	U5 DANGER LCIB
 
+## Configuring report list titles
+
+The CHT shows the name of the subject of the report in the reports list by default. This can be overridden by configuring the `subject_key` property with a translation key in the form document, for example:
+
+```json
+  "internalId": "signal",
+  "title": "Signal signoff",
+  "subject_key": "signal.list.subject",
+```
+
+The translation uses a summary of the report as the evaluation context so you can include report fields in your value, for example: `Case registration {{case_id}}`.
```

Comment:
Is there a definition of "a summary of the report" somewhere?

Reply:
I don't think there is. Do you think it's needed?

Reply:
Probably ?, since the "summary" is actually defined in a) a view or b) some code that should be the same as the view, I guess the fields are a surprise to anyone not familiar with those. 

I, personally, had to look it up to see what the summary was.

Reply:
You're right. Thanks for pulling me up on this!